shinatra.sh

```diff
@@ -1,5 +1,2 @@
 #!/bin/bash
-RESPONSE="HTTP/1.1 200 OK\r\nConnection: keep-alive\r\n\r\n${2:-"OK"}\r\n"
-while { echo -en "$RESPONSE"; } | nc -l "${1:-8080}"; do
-  echo "================================================"
-done
+python -m SimpleHTTPServer "${1:-8080}"
```

Comment:
FWIW `python` should be python 2, as suggested by PEP 394
(except for, again, Arch :) )